### PR TITLE
[lzop] Add license file as %license. JB#55991

### PR DESCRIPTION
**Pull request by pvuorela**
@Tomin1 @mlehtima

### CODE REVIEW

File: rpm/lzop.spec [hunk 1]
```diff
@@ -39,7 +37,7 @@ lzop documentation. Man pages and html docs.
 export CPPFLAGS="-I/usr/include/lzo"
 export LDFLAGS="-L%{_libdir}"
 %configure
-%__make %{?jobs:-j%{jobs}}
+%make_build
 
 %install
 %makeinstall
```
Comment (Tomin1) [inline rpm/lzop.spec:43]: Just an observation: so we have both %make_install and %makeinstall? :thinking:

**Review by Tomin1 — APPROVED** [review]
Looks fine, though that `%clean` section could also go away.

File: rpm/lzop.spec (hunk 1 above)
Comment (pvuorela) [reply rpm/lzop.spec:43]: Hm, no idea really.

Comment (mlehtima) [reply rpm/lzop.spec:43]: %makeinstall is old naming and should not be used in general, it works though.

Comment (Thaodan) [reply rpm/lzop.spec:43]: > Just an observation: so we have both %make_install and %makeinstall? thinking

https://github.com/rpm-software-management/rpm/commit/0fddb3cbd80fc763ecacfea5b82631f7693915c2

Comment (Tomin1) [reply rpm/lzop.spec:43]: Ok, I get it. We should fix those on sight.

**Pull request merged**